src/sentry/integrations/github/repository.py

```diff
@@ -65,25 +36,25 @@ def validate_config(self, organization, config):
         return config
         ```
         """
-        if config.get('name') and config.get('installation'):
+        if config.get('identifier') and config.get('installation'):
             integration = Integration.objects.get(
                 id=config['installation'], organizations=organization)
             installation = integration.get_installation(organization.id)
             client = installation.get_client()
 
-            repo = self._validate_repo(client, installation, config['name'])
+            repo = self._validate_repo(client, installation, config['identifier'])
             config['external_id'] = six.text_type(repo['id'])
             config['integration_id'] = integration.id
```

Comment:
This is probably more of a nit than anything, but isn't `installation` also `integration_id`? did we need to store it twice? Or am I misremembering that?

Reply:
not sure what you mean by storing it twice, can you clarify?

Reply:
I was thinking installation was the same as integration_id. I think it is, but the repo code expects integration. So that's fine.